projects/core/src/lib/services/map-pdf/map-pdf.service.ts

```diff
@@ -83,11 +83,11 @@ export class MapPdfService {
   private addDateTime(doc: jsPDF, width: number, height: number) {
     const text = $localize `Gemaakt op `;
     const date = text + new Intl.DateTimeFormat('nl-NL', { dateStyle: 'full', timeStyle: 'medium'}).format(new Date());
```

Comment:
Instead of using hard-coded nl-NL, it would be better to use the LOCALE_ID injection token to get the current locale so this is also changed when changing the locale in an application:

`@Inject(LOCALE_ID) public locale: string,`